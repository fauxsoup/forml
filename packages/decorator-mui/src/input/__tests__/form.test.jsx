import FormComponent from "../form";
import React from "react";
import { render } from "@testing-library/react";

describe("renders", function() {
    let form;

    beforeEach(function() {
        form = { type: "text" };
    });

    test('consistently', function() {
        const { container } = render(
            <FormComponent form={form} />
        )

        expect(container).toMatchSnapshot();
    })
});
