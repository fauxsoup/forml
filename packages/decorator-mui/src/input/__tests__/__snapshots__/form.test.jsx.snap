// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders consistently 1`] = `
<div>
  <div
    class="MuiInputBase-root MuiInput-root MuiInput-underline"
    form="[object Object]"
  >
    <input
      aria-invalid="false"
      class="MuiInputBase-input MuiInput-input"
      type="text"
      value=""
    />
  </div>
</div>
`;
